pkg/controller/kibana/config_reconcile.go

```diff
@@ -23,28 +23,43 @@ import (
 
 // Constants to use for the config files in a Kibana pod.
 const (
-	VolumeName        = "config"
-	VolumeMountPath   = "/usr/share/kibana/" + VolumeName
+	ConfigVolumeName                   = "elastic-internal-kibana-config-local"
```

Comment:
a few lines of comments explaining the general logic and why we have several volumes would be nice